src/TestFSharp/TestFSharp.fsproj

```diff
@@ -32,6 +32,29 @@
     <WarningLevel>3</WarningLevel>
     <DocumentationFile>bin\Release\TestFSharp.XML</DocumentationFile>
   </PropertyGroup>
+  <!--BEGIN: Workaround allowing the use of the FSharp.Compiler.Tools package from within Visual Studio-->
+  <Choose>
+    <When Condition="Exists('..\packages\FSharp.Compiler.Tools.4.0.0.1\tools\Microsoft.FSharp.Targets')">
+      <PropertyGroup>
+        <FSharpTargets>..\packages\FSharp.Compiler.Tools.4.0.0.1\tools\Microsoft.FSharp.Targets</FSharpTargets>
+        <PackagesRestored>true</PackagesRestored>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <FSharpTargets>$(MSBuildBinPath)\Microsoft.Common.targets</FSharpTargets>
+        <PackagesRestored>false</PackagesRestored>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+  <Target Name="CreateManifestResourceNames" Condition="'@(EmbeddedResource)' != '' And '$(PackagesRestored)' == 'false'">
+  </Target>
+  <Target Name="CoreCompile" Condition="'$(PackagesRestored)' == 'false'" Inputs="$(MSBuildAllProjects);&#xD;&#xA;            @(CompileBefore);&#xD;&#xA;            @(Compile);                               &#xD;&#xA;            @(CompileAfter);&#xD;&#xA;            @(_CoreCompileResourceInputs);&#xD;&#xA;            @(ManifestNonResxWithNoCultureOnDisk);&#xD;&#xA;            $(ApplicationIcon);&#xD;&#xA;            $(AssemblyOriginatorKeyFile);&#xD;&#xA;            @(ReferencePath);&#xD;&#xA;            @(CompiledLicenseFile);&#xD;&#xA;            @(EmbeddedDocumentation); &#xD;&#xA;            $(Win32Resource);&#xD;&#xA;            $(Win32Manifest);&#xD;&#xA;            @(CustomAdditionalCompileInputs);&#xD;&#xA;            $(VersionFile);&#xD;&#xA;            $(KeyOriginatorFile)" Outputs="@(DocFileItem);&#xD;&#xA;              @(IntermediateAssembly);&#xD;&#xA;              @(_DebugSymbolsIntermediatePath);                 &#xD;&#xA;              $(NonExistentFile);&#xD;&#xA;              @(CustomAdditionalCompileOutputs)">
```

Comment:
I think that VS xml encoded the 0x0d, 0x0a CRLF characters properly. It causing no problems just XML conform ;-)